testdata/scripts/node/validate/valid.txtar

```diff
@@ -502,6 +502,7 @@ EnforceRepeatableRead = true
 DeathDeclarationDelay = '1m0s'
 NewHeadsPollInterval = '0s'
 VerifyChainID = true
+ExternalRequestMaxResponseSize = 0
```

Comment:
Nit/ it's a bit long - could we drop a word?
```suggestion
ExternalMaxResponseSize = 0
```
Alternatively, do we expect more External fields in the future? Maybe a subsection in that case?

Reply:
Renaming this field require changes to the already merged chainlink-evm. This PR contains a fix to the breaking changes introduced in chainlink-common that have already been merged.
Could we merge this PR as is and do the renaming as a follow-up?